<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1103</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>987</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-50</number>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	border-radius:0px;
	padding:10 px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 59, 90);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="3" column="0">
     <widget class="Line" name="line_5">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="Line" name="line_2">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="image5">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30;
padding:25px;
color: white;</string>
      </property>
      <property name="text">
       <string>Loading Image...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 44, 68);
background-color: rgb(42, 46, 70);
color: rgb(97, 97, 97);</string>
      </property>
      <property name="text">
       <string>Created by Aayush and Abhishek</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="Line" name="line_7">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="image6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>176</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30;
padding:25px;
color: white;</string>
      </property>
      <property name="text">
       <string>Loading Image...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="image1">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30;
padding:25px;
color: white;</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Loading Image...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="Line" name="line_4">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="image2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>176</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30;
padding:25px;
color: white;</string>
      </property>
      <property name="text">
       <string>Loading Image...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="image3">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30;
padding:25px;
color: white;</string>
      </property>
      <property name="text">
       <string>Loading Image...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="Line" name="line_8">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lname2">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="name2">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
        </property>
        <property name="text">
         <string>See Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lname6">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="name6">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
        </property>
        <property name="text">
         <string>See Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lname1">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="name1">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
        </property>
        <property name="text">
         <string>See Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="lname5">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="name5">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
        </property>
        <property name="text">
         <string>See Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <widget class="Line" name="line_9">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="Line" name="line">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="lname3">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="name3">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
        </property>
        <property name="text">
         <string>See Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <widget class="Line" name="line_6">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="Line" name="line_3">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="image4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>176</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30;
padding:25px;
color: white;</string>
      </property>
      <property name="text">
       <string>Loading Image...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="Line" name="line_10">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QWidget" name="titleBarFrame" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 44, 68);
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">border: 0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minimizeButton">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Minimize</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maximizeButton">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Maximize</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Close</string>
         </property>
         <property name="whatsThis">
          <string>Close</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="Line" name="line_12">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lname4">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="name4">
        <property name="font">
         <font>
          <family>Oswald</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
        </property>
        <property name="text">
         <string>See Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="Line" name="line_11">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="Line" name="line_13">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="Line" name="line_14">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="Line" name="line_15">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="Line" name="line_16">
      <property name="font">
       <font>
        <family>Oswald</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
